Userland/Libraries/LibJS/Bytecode/PassManager.h

```diff
@@ -136,6 +136,15 @@ class DumpCFG : public Pass {
     FILE* m_file { nullptr };
 };
 
+class LoadElimination : public Pass {
+public:
+    LoadElimination() = default;
+    ~LoadElimination() override = default;
```

Comment:
`virtual`

Reply:
This is copied from another pass and renamed, but sure

Reply:
Rechecking, all passes only declare as override and just imply virtual

Reply:
It's against our coding style, and you PR'd this code - I don't care if it was there already. :shrug:
https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#overriding-virtual-methods